test/commands/scanner/rule/add.test.ts

```diff
@@ -10,12 +10,9 @@ Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/sfdx-scanner', 'add');
 
 describe('scanner:rule:add', () => {
-	// Reset our controller since we are using alternate file locations
-	before(() => TestOverrides.initializeTestSetup());
-
 	describe('E2E', () => {
 		const myLanguage = 'apex';
-		it('Test Case: Adding individual JARs', () => {
```

Comment:
I previously switched these from `describe` to `it`, but that was a mistake, they need to be describe.